src/selectors/generateChannelLink.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Returns a link depending on the channel
+ * Depending on pageUrl, will either append to existing query params or create a new query
+ * @param {string} pageURL - Page url.
+ * @param {string} integrationName - Will append channel depending on integration name.
+ */
+
+export function generateChannelLink(pageURL, integrationName) {
+  let channelName;
+  switch (integrationName) {
+    case 'Magento':
+      channelName = 'Integration: Magento 2';
+      break;
+    case 'cpanel':
+      channelName = 'Integration: CPanel';
+      break;
+    case 'wordpress':
+      channelName = 'Integration: WordPress';
+      break;
+    default:
+      channelName = '';
+  }
```

Comment:
`generateChannelLink("url.com", "somethingisnotright")` returns `"url.com?channel="`

It would be better to not generate the channel if it's going to be `''`. But I think it may be better to put something there. It may allows us figure if something is wrong or if someone else has built there own custom integration using this code. Both are fine information I guess.